src/exercise/04.md

```diff
@@ -37,7 +37,10 @@ returns the React element. Then you can interpolate a call to that function in
 your JSX.
 
 ```jsx
-<div>{message('Hello World')}</div>
+<div className="container">
+  {message('Hello World')}
+  {message('Goodbye World')}
+</div>
 ```
```

Comment:
Hmm... Unfortunately this is the solution. I normally like to be a little less obvious.

This is a problem for sure, but I'm not sure that sharing the final version is the best solution 😬

Reply:
Fair. I am not sure what the best solution, just had a friend stuggle on this step.

Reply:
He suggests using a different className there, such that it becomes more clear, for example, 

```
<div className = “testName”> 
  {message('Hello World')} 
</div>
```

Reply:
Maybe we could do: `<span className="example">{message('Hello World')}</span>` to make sure it's clear that this is just an example. What do you think?

Reply:
Yes, it's better :)

Reply:
Cool, would you like to make a new PR for that?